### ISSUE TITLE
[AVT] : Tabs component : Multiple 'navigation' sections or landmarks must have unique labels specified with 'aria-label' or 'aria-labelledby'.

### ISSUE THREAD

**Issue by PAnilReddy** [carbon-design-system/carbon#6124]
Cloudpal version
"@console/pal": "^1.134.1",

Tabs dont have an option to add aria-label/aria-labelledby. We need this option as when there are 2 tab components with role=navigation, then we start getting this violation since there is no way to specify its uniqueness.

![image](https://user-images.githubusercontent.com/44162180/82646564-6d0ba900-9c32-11ea-85c7-fcd68d823687.png)

**Comment by joshblack** [comment]
Hi there @PAnilReddy! 👋 You can pass in the `aria-label` or `aria-labelledy` prop to specify this landmark for your specific application 👍 

Reference: https://codesandbox.io/s/distracted-bartik-nry3q?file=/src/index.js

**Comment by PAnilReddy** [comment]
I did pass the variable aria-label but it does not take effect.
I inspected the element as well but I cannot see that attribute on the dom

**Comment by joshblack** [comment]
Got it, thanks for sharing! Could you provide some steps to reproduce the issue through a codesandbox from the issue template?

**Comment by PAnilReddy** [comment]
Hey got it working. We have given ariaLabel which we have taken it checking the tab code but dint work. We changed it to aria-label and it works. Not sure if it has to work for ariaLabel as well because for other components this works.
Thank you

**Comment by carmacleod** [comment]
Unfortunately, adding aria-label is the wrong fix. The right fix is to delete role="navigation". Please see: https://github.com/carbon-design-system/carbon/issues/6931
